packages/mdc-chips/chip-set/foundation.js

```diff
@@ -45,6 +53,42 @@ class MDCChipSetFoundation extends MDCFoundation {
    */
   constructor(adapter) {
     super(Object.assign(MDCChipSetFoundation.defaultAdapter, adapter));
+
+    /** 
+     * The active chips in the set. Only used for choice chip set or filter chip set.
+     * @private {!Array<!MDCChip>}
+     */
+    this.activeChips_ = [];
+    /** @private {function(!Event): undefined} */
+    this.chipInteractionHandler_ = (evt) => this.handleChipInteraction_(evt);
+  }
+
+  init() {
+    this.adapter_.registerEventHandler(MDCChipFoundation.strings.INTERACTION_EVENT, this.chipInteractionHandler_);
+  }
+
+  destroy() {
+    this.adapter_.deregisterEventHandler(MDCChipFoundation.strings.INTERACTION_EVENT, this.chipInteractionHandler_);
+  }
+
+  /**
+   * Handles a chip interaction event
+   * @param {!Object} evt
+   * @private
+   */
+  handleChipInteraction_(evt) {
+    const {chip} = evt.detail;
+    if (this.adapter_.hasClass(cssClasses.CHOICE)) {
```

Comment:
Can we invert this `if` condition and bail out early if the root element does _not_ have the `--choice` class?

That'll reduce nesting and make the code a bit easier to read.

Reply:
This is preemptively structured this way to account for the other chip set variants coming soon, but I'll invert it for now and change it in a later PR.